src/Microsoft.SqlTools.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -293,15 +295,22 @@ public void InitializeService(ServiceHost serviceHost, SqlToolsContext context)
             }
         }
 
-// turn off this code until needed (10/28/2016)
-#if false
-        private static async Task HandleDefinitionRequest(
-            TextDocumentPosition textDocumentPosition,
-            RequestContext<Location[]> requestContext)
+        internal static async Task HandleDefinitionRequest(TextDocumentPosition textDocumentPosition, RequestContext<Location[]> requestContext)
```

Comment:
Yes, we should add the check here. There are no options for definition per se but I'll add the check for is intellisense enabled